core/services/chainlink/application.go

```diff
@@ -297,10 +301,37 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 	promReporter := promreporter.NewPromReporter(db.DB, globalLogger)
 	srvcs = append(srvcs, promReporter)
 
+	// Initialize Local Users ORM and Authentication Provider specified in config
+	// LocalAdminUsersORM is initialized and required regardless of separate Authentication Provider
+	var localAdminUsersORM sessions.LocalAdminUsersORM
+	localAdminUsersORM = localauth.NewORM(db, cfg.WebServer().SessionTimeout().Duration(), globalLogger, cfg.Database(), auditLogger)
+
+	// Initialize Sessions ORM based on environment configured authenticator
+	// localDB auth or remote LDAP auth
+	authMethod := cfg.WebServer().AuthenticationMethod()
+	var authenticationProvider sessions.AuthenticationProvider
+	var sessionReaper utils.SleeperTask
+
+	switch authMethod {
+	case string(sessions.LDAPAuth):
```

Comment:
Would it be a litter cleaner to convert the variable first?
```suggestion
	switch session.AuthenticationProviderName(authMethod) {
	case sessions.LDAPAuth:
```

Reply:
Yeah that does read cleaner, updated